packages/seeder/src/seeder.js

```diff
@@ -170,7 +170,7 @@ class Seeder extends EventEmitter {
       // get namespace
       const store = this._store.namespace(key)
       // Create drive
-      drive = new Drive(key, store, { size, logger: this._logger })
+      drive = new Drive(key, store, { size, logger: this._logger }, this._opts.secret)
```

Comment:
What does the secret do when passed to the `Drive`? I'm not sure I understand the functionality here.

Reply:
internally we called a function to get the peers information, this information is obtained per drive (hyperdrive), so we end up hashing the output of that function.